{{cookiecutter.repo_name}}/setup.py

```diff
@@ -16,15 +16,15 @@
     with open("README.md", "r") as handle:
         long_description = handle.read()
 except:
-    long_description = "\n".join(short_description[2:])
```

Comment:
This would have generated an IndexError with the test case if README.md were ever missing.

Reply:
Good catch, but I say let it raise an exception. Project descriptions should have more details than 1 line. I think a more graceful exception would be good, and one which won't auto-fail due to a typo here.

Reply:
The `short_description` is now explicitly one line with this change. Do you prefer to leave long_description blank (as with the current commit) or to allow the potential `open()` exception to propagate?

Reply:
Lets leave it as a known bug for now and fix it in #136. Its never come up before because I'm operating on the assumption the README file should always be present, so taking a bit longer to fix it shouldn't harm anything.

Reply:
Can you please clarify?
Do you want to
* leave the PR as it is?
* push a commit to revert this change and leave a substantial logic error to be resolved in another PR?
* push a commit to remove the `except` block?

Reply:
No specific edit has been requested and the thread has been idle for 10 days. Marking "resolved".